apps/edxapp/templates/services/cms/job_01_internationalization.yml.j2

```
@@ -27,7 +27,7 @@ spec:
         - name: edxapp-cms-internationalization-{{ job_stamp }}
           # We point to a local registry image build for this "live" image (see
           # ImageStream and BuildConfig templates)
-          image: "{{ internal_docker_registry }}/{{ project_name }}/edxapp-cms:{{ edxapp_image_tag }}-live"
+          image: "{{ internal_docker_registry }}/{{ project_name }}/{{ image_name(service_variant, edxapp_image_tag, edxapp_theme_tag) }}"
```

`service_variant` variable is not defined here, should be:

```suggestion
          image: "{{ internal_docker_registry }}/{{ project_name }}/{{ image_name('cms', edxapp_image_tag, edxapp_theme_tag) }}"
```

---

The same applies for all other job templates modifications.